en/development/configuration.rst

```diff
@@ -51,7 +51,7 @@ datasource
 persistent
     Whether or not to use a persistent connection to the database. When using
     SQLServer you should not enable persistent connections as it causes
-    difficult to diagnose crashes.
+    difficulty to diagnose crashes.
```

Comment:
I see.  So "difficult to diagnose" is like an adjective to describe "crashes", and "it causes crashes".  It makes sense.  English is not my native language either.  I started learning English from grammar, not like native speakers started with using it.

I'd better restore as it was.  I just wanted to pick up apparent errors in original text that may hinder the correct understanding of how to use CakePHP.  Fine tuning English is not what I should go into.